src/raw/unix.rs

```diff
@@ -394,8 +395,6 @@ impl Socket {
     }
 
     ///Sets I/O parameters of socket.
-    ///

```

Comment:
On Unix the `.ioctl(…)` method simply calls `ioctl(2)`, no point in stating the obvious IMHO. (Also if have to you ioctl for some reason, then you tend to know why…)
Also the Windows docs retain the mention of the `ioctlsocket` function.